content/guides/getstarted/rust-to-solana.md

```diff
@@ -0,0 +1,186 @@
+
+However, if the crate used simply depends on `rand` but does not actually generate random numbers, then it is possible to work around this by adding the following to the program’s Cargo.toml:
+
+```toml
+[dependencies]
+getrandom = { version = "0.1.14", features = ["dummy"] }
+```
+
+### Macro Changes
+Several standard macros have been replaced or have modified behavior to be aware of. First, the `println!` has been replaced with the computationally simpler `msg!` macro. The `msg!` macro outputs to the program logs, and it can be used as follows:
+
+```rust 
+msg!(“Your message”);
+msg!(0_64, 1_64, 2_64);
+msg!(“Your variable: {:?}”, variable);
+```
+
+The `panic!`, `assert!`, and any internal panics are also output to the program logs by default. However, this can be modified with a custom panic handler.
+
+## Compute Budget 
+As a Rust developer, efficient computing is nothing new. What may be different is that in Solana, each transaction has a fixed [compute budget](https://solana.com/docs/core/runtime#compute-budget) that it must not surpass. When transactions exceed the compute budget, they are halted and return an error.
+
+Programs can access the number of remaining compute units via the `sol_remaining_compute_units` system call, and can log the remaining number of compute units with `sol_log_compute_units`.
+
+## Learning the Development Environment and Frameworks
+While the Solana CLI and the `solana_program` crate are all that is needed to get started, there are a couple of helpful tools which can accelerate learning.
+
+<div align="center">
+    <img src="./rust-to-solana-img/network-diagram.png" alt="Network Diagram" style="max-width: 80%; height: auto;"/>
+</div>
+
+### Solana Playground 
+The [Solana Playground](https://beta.solpg.io/) is a browser-based IDE that allows developers to develop and deploy Solana programs.
+
+<div align="center">
+    <img src="./rust-to-solana-img/solana-playground.png" alt="Solana Playground" style="max-width: 80%; height: auto;"/>
+</div>
+
+It’s the easiest way to begin developing with Solana, and it supports building, testing, and deploying Solana Rust programs. Additionally, a number of built-in tutorials are available to guide learning.
+
+The Solana Playground allows developers to get started immediately without setting up a local environment.
+
+### Using Anchor 
+[Anchor](https://www.anchor-lang.com/) is a framework that seeks to accelerate the building of secure Solana programs. It can help by handling standard boilerplate code, speeding up the development cycle. Additionally, it provides some security checks by default, making Solana programs more secure.
+
+To create a new program, simply create a new Anchor project in the Solana playground. 
+
+Alternatively, install the Anchor CLI locally, and then use `anchor init <project-name>` to create a new Anchor project.
```

Comment:
Maybe add a link to the anchor installation here? [install the Anchor CLI locally](https://www.anchor-lang.com/docs/installation)